java/src/test/java/ai/onnxruntime/InferenceTest.java

```diff
@@ -57,76 +47,56 @@
 /** Tests for the onnx-runtime Java interface. */
 public class InferenceTest {
   private static final Logger logger = Logger.getLogger(InferenceTest.class.getName());
-  private static final Pattern LOAD_PATTERN = Pattern.compile("[,\\[\\] ]");
 
   private static final String propertiesFile = "Properties.txt";
 
   private static final Pattern inputPBPattern = Pattern.compile("input_*.pb");
   private static final Pattern outputPBPattern = Pattern.compile("output_*.pb");
 
-  private static Path getResourcePath(String path) {
-    return new File(InferenceTest.class.getResource(path).getFile()).toPath();
-  }
-
-  @Test
-  public void repeatedCloseTest() throws OrtException {
-    Logger.getLogger(OrtEnvironment.class.getName()).setLevel(Level.SEVERE);
-    OrtEnvironment env = OrtEnvironment.getEnvironment("repeatedCloseTest");
-    try (OrtEnvironment otherEnv = OrtEnvironment.getEnvironment()) {
-      assertFalse(otherEnv.isClosed());
-    }
-    assertFalse(env.isClosed());
-    env.close();
-    assertTrue(env.isClosed());
-  }
+  private static final OrtEnvironment env = OrtEnvironment.getEnvironment();
```

Comment:
> env = OrtEnvironment.getEnvironment(); [](http://example.com/codeflow?start=37&length=39)

Is there a test that checks that repeated calls return the same instance?

Reply:
No, I've not added an explicit test for that. I can do.